playbooks/maas-ceph-rgw.yml

```diff
@@ -0,0 +1,70 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Gather facts
+  hosts: rgws
+  gather_facts: "{{ gather_facts | default(true) }}"
+  tasks:
+    - include: "common-tasks/maas_excluded_regex.yml"
+    - name: Set the current group
+      set_fact:
+        maas_current_group: rgws
+  
+  tags:
+    - maas-ceph-rgws
+
+- include: maas-ceph-raxmon.yml
+
+- name: Install checks for ceph rgws
+  hosts: rgws
+  gather_facts: false
+  pre_tasks:
+    - name: Write Ceph monitoring client key to file
+      copy:
+        content: "{{ [hostvars[groups['mons'][0]]['ceph_raxmon_client']['stdout'], '\n'] | join('') }}"
+        dest: "/etc/ceph/ceph.client.raxmon.keyring"
+      delegate_to: "{{ physical_host | default(ansible_host) }}"
+
+    - name: Write Ceph monitoring client key to file within legacy rpc hosts
+      copy:
+        content: "{{ [hostvars[groups['mons'][0]]['ceph_raxmon_client']['stdout'], '\n'] | join('') }}"
```

Comment:
Instead of this being "[hostvars[groups['mons'][0]]"  shouldn't it be "[hostvars[groups['rgws']]